### PR TITLE
Feature/backend 1879

### PR DESCRIPTION
**Pull request by breath103**
Currently, when error occurs corgi format error without stackTrace since it can leak internal structure of system.

in order to debug error easily, 
1) if you set **CORGI_ERROR_PASSWORD**
2) error response would have "metadata" which is encrypted stackTrace + error message
3) you can decrypt this with same key

### CODE REVIEW

**Review by deerFried — APPROVED** [review]
👍

**Review by syeutyu — APPROVED** [review]

File: src/error_response.ts [hunk 1]
```diff
@@ -15,3 +28,52 @@ export interface StandardErrorResponseBody {
...
+    } else {
+      // For Non-Standard error,
+      return {
+        code: error.name,
+        message: error.message,
+        metadata: (() => {
+          if (this.password) {
+            const cipher = crypto.createCipher(CipherAlgorithm, this.password)
+            return [
+              cipher.update(JSON.stringify({
```
Comment (mooyoul) [inline src/error_response.ts]: const payload = JSON.stringify(obj);

return Buffer.concat([
  cipher.update(payload),
  cipher.final(),
]).toString('hex');

로 쓰는게 어떨까요?

**Review by mooyoul — CHANGES_REQUESTED** [review]

File: src/error_response.ts [hunk 2]
```diff
@@ -15,3 +28,52 @@ export interface StandardErrorResponseBody {
...
+        metadata: error.options.metadata,
+      };
+    } else {
+      // For Non-Standard error,
+      return {
+        code: error.name,
+        message: error.message,
+        metadata: (() => {
+          if (this.password) {
+            const cipher = crypto.createCipher(CipherAlgorithm, this.password)
```
[the remark this answers is not in the document]
Comment (breath103) [reply src/error_response.ts]: createCipheriv 로 하려면 HMAC key도 만들어야 해서 너무 복잡하긴 한데.. 검토는 해볼께요

**Comment by breath103** [comment]
HMAC 까지는 안하고 IV 키만 추가해서 끝냈습니다.

**Pull request merged**